### PR TITLE
Hlm 1174 UI static screen beneficiaries

### PR DESCRIPTION
**Pull request by naveen-egov**

### CODE REVIEW

**Comment by naveen-egov** [comment]
Screenshot 2022-12-21 at 8 35 34 AM
Screenshot 2022-12-21 at 8 38 10 AM


Fixed the test cases.

File: apps/health_campaign_field_worker_app/lib/router/app_router.dart [hunk 1]
```diff
@@ -19,6 +20,7 @@ part 'app_router.gr.dart';
       children: [
         AutoRoute(page: HomePage, path: ''),
         AutoRoute(page: FormsPage, path: 'forms/:pageName'),
+        AutoRoute(page: SearchBeneficiaryPage, path: 'search-benificary'),
```
Comment (egov-joy) [inline apps/health_campaign_field_worker_app/lib/router/app_router.dart:23]: typo: benificiary should be beneficiary

File: apps/health_campaign_field_worker_app/test/widgets/benificary_inbox_card.test.dart [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import 'package:health_campaign_field_worker_app/models/beneficiary_statistics/beneficiary_statistics_model.dart';
+import 'package:health_campaign_field_worker_app/widgets/beneficiary/beneficiary_%20statistics_card.dart';
+import '../widget_app.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group(
+    'BenificiaryInboxCard Test',
```
Comment (egov-joy) [inline apps/health_campaign_field_worker_app/test/widgets/benificary_inbox_card.test.dart:8]: typo: benificiary should be beneficiary. Also in filename.

File: apps/health_campaign_field_worker_app/test/widgets/benificiary_card_test.dart [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import 'package:health_campaign_field_worker_app/widgets/beneficiary/beneficiary_card.dart';
```
Comment (egov-joy) [inline apps/health_campaign_field_worker_app/test/widgets/benificiary_card_test.dart:1]: typo: benificiary should be beneficiary in filename

**Review by egov-joy — COMMENTED** [review]
Once #16 is merged, let's also update this PR to use localized text everywhere instead of fixed text before merging.

File: apps/health_campaign_field_worker_app/lib/router/app_router.dart (hunk 1 above)
Comment (egov-joy) [reply apps/health_campaign_field_worker_app/lib/router/app_router.dart]: @naveen-egov still has a missing 'i'

File: apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart [hunk 4]
```diff
@@ -0,0 +1,23 @@
+class i18 {
+  static Common common = const Common();
+  static SearchBeneficiary searchBeneficiary = const SearchBeneficiary();
+}
+
+class Common {
+  const Common();
+  String get coreCommonContinue => 'CORE_COMMON_CONTINUE';
+  String get coreCommonAge => 'CORE_COMMON_AGE';
+  String get coreCommonGender => 'CORE_COMMOM_GENDER';
```
Comment (egov-joy) [inline apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart:10]: COMMOM -> COMMON

File: apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+class SearchBeneficiary {
+  const SearchBeneficiary();
+  String get listOfHouseholds => 'LIST_OF_HOUSE_HOLDS';
```
Comment (egov-joy) [inline apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart:15]: household is one word.. should be LIST_OF_HOUSEHOLDS

File: apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  String get noOfHouseholdsRegistered => 'NO_OF_HOUSE_HOLDS_REGISTERED';
```
Comment (egov-joy) [inline apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart:16]: NO_OF_HOUSEHOLDS_REGISTERED

File: apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  String get noOfBedetsDelivered => 'NO_OF_BEDETSDELIVERED';
```
Comment (egov-joy) [inline apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart:17]: noOfBedetsDelivered -> noOfBednetsDelivered
NO_OF_BEDETSDELIVERED -> NO_OF_BEDNETS_DELIVERED

File: apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+  String get searchHintText => 'SEARCH_HINT_TEXT';
+  String get beneficiaryInfoDescription => 'BENEFICIARY_INFO_DESCRIPTION';
+  String get beneficiaryInfoTitle => 'BENEFICIARY_INFO_TITLE';
+  String get actionLabel => 'ACTION_LABEL';
```
Comment (egov-joy) [inline apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart:21]: Shouldn't we have screen specific keys and values here so it is easy to distinguish the action label?

File: apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+  String get coreCommonAge => 'CORE_COMMON_AGE';
+  String get coreCommonGender => 'CORE_COMMOM_GENDER';
+}
+
+class SearchBeneficiary {
+  const SearchBeneficiary();
+  String get listOfHouseholds => 'LIST_OF_HOUSE_HOLDS';
+  String get noOfHouseholdsRegistered => 'NO_OF_HOUSE_HOLDS_REGISTERED';
+  String get noOfBedetsDelivered => 'NO_OF_BEDETSDELIVERED';
+  String get searchHintText => 'SEARCH_HINT_TEXT';
```
Comment (egov-joy) [inline apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart:18]: Is this for the search by name of head of household search? We should have a more specific key if so.

File: apps/health_campaign_field_worker_app/lib/utils/18KeyConstants.dart [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+class Login {
+  const Login();
+  String get labelText => 'LOGIN_LABEL_TEXT';
+  String get userIdPlaceholder => 'USER_ID_PLACEHOLDER';
+  String get passwordPlaceholder => 'PASSWORD_PLACEHOLDER';
+  String get actionLabel => 'LOGIN_ACTION_LABEL';
+}
+
+class ForgetPassword {
```
Comment (egov-joy) [inline apps/health_campaign_field_worker_app/lib/utils/18KeyConstants.dart:23]: lines 23-28: Forget -> Forgot

**Review by egov-joy — APPROVED** [review]

**Pull request merged**